examples/qualcomm/scripts/ptq_mobilebert_fine_tune.py

```diff
@@ -0,0 +1,314 @@
+# Copyright (c) Qualcomm Innovation Center, Inc.
+# All rights reserved
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
```

Comment:
So this is not ptq fine tune, right? YOu are further training. Plus why do you need this?

Reply:
No, it just fine tune for classifier of the mobilebert multi-class text classification

Reply:
Right, but fine tuning is via further training, correct?

Reply:
Yes, that right. Because some weights of MobileBertForSequenceClassification were not initialized from the model checkpoint at google/mobilebert-uncased. Therefore. we should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.